.env

```diff
@@ -0,0 +1,4 @@
+GOOGLE_CLIENT_ID=630192318443-askf6t4pcvsrbd65u90fr5sqln660vtd.apps.googleusercontent.com
```

Comment:
its generally not best practice to commit your .env cause it could contain secrets others could steal and exploit